doc/links.txt

```diff
@@ -28,6 +28,8 @@
 
 .. _`ZBOSS API documentation`:
 .. _`ZBOSS stack release notes`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/index.html
+.. _`certification policy`: https://zigbeealliance.org/wp-content/uploads/2021/04/07-4842-13-Zigbee-certification-policy.pdf
+.. _`Zigbee Compliant Platforms page`: https://zigbeealliance.org/classic-platform-search/#zcp/productdetails5/609a51c246b7d5001b613256/
```

Comment:
This link will point out only to 840 certification, while we expect to get this week also certification for 833 
Maybe link https://zigbeealliance.org/classic-platform-search/#zcp/?view_197_search=nordic%20Connect%20SDK&view_197_page=1 would be better?